### DISCUSSION TITLE
Player gets rare resizing

### DISCUSSION THREAD

**Discussion by michelson** [katspaugh/wavesurfer.js#2944] [General]
I have some weird error using version 7 beta 11, where the player gets resized
not sure what could cause this, as it works ok on some pages and some not; probably is something related to the responsive resizing, but I do not have more context, maybe I could get some lights here so I could debug this. PLease see the attached video.

https://github.com/katspaugh/wavesurfer.js/assets/11976/7577295b-e333-4b1e-910c-2f7de4010266

Thanks in advance for the help; please let me know where I can send you the coffee!

**Comment by katspaugh** [comment]
I made a similar example with flexbox, and it doesn't jump like on your video: https://wavesurfer-js.org/examples/#d1e2980dcf920a636b79f554681a52d5

What are the options you're passing to wavesurfer?

**Reply by michelson** [reply]
Hi @katspaugh , it seems that the `minPxPerSec: 2,` was the option that makes the player jump, removing it did the trick, thanks for the example. 👍

**Reply by katspaugh** [reply]
Gotcha, I see the jumping with minPxPerSec too in that example, will make an issue for it.
